web/src/main/resources/static/js/main.js

```diff
@@ -18,9 +30,9 @@ $(document).ready(function(){
       $('#tree-tab').click(function (e) {
```

Comment:
Note that this does not work: http://ONCOTREE_URL/?search_term=SEZS, but that problem was not introduced here.  You have to use: http://ONCOTREE_URL/#/home?search_term=SEZS, which is not ideal.  We should fix that another time.